### ISSUE TITLE
[BUG] Generate WebP thumbnails instead of PNG thumbnails to save space, memory, and page load time

### ISSUE THREAD

**Issue by adam-ah** [paperless-ngx/paperless-ngx#1124]
### Description

# The problem

<img width="698" alt="image" src="https://user-images.githubusercontent.com/3745745/173000963-ce380717-0942-4bf7-896d-d875d2463c2e.png">

**Loading the 6 simple thumbnails add over 7mb to the pageload**

- Problem 1: Thumbnails are full sized images (rather than actual thumbnails) and stored in a inefficient, lossless PNG format. 
- Problem 2: All the thumbnails are loaded when the `/documents` page load.

A typical thumbnail of an invoice
- Current size in PNG format: 2,720,732 bytes (100%)
- When (manually) converted to JPG: 280,082 bytes (10.3%)
- When (manually) converted to WebP: 89,806 bytes (3.3%)

# Why is it important?

The `/documents` is probably the most requested page. After the fix, the
- load-time with many documents (and their thumbnails) would be substantially faster both locally and over remote access
- storage requirement on the server would be less
- memory requirement in the browser would be much lower

# How hard is it to fix?

Really simple: ImageMagick already support converting to WebP so the two simple changes would be
- Change the filename in the `convert` command from `.png` to `.webp`
- Change the filename that is retrieved by the `/thumb` url

### Steps to reproduce

Load `/documents`

### Paperless-ngx version

1.7.1

### Host OS

Ubuntu

### Installation method

Docker

**Comment by shamoon** [comment]
Yes! This is a great idea I keep forgetting to circle back to, thanks for the reminder. I can look into this.

**Comment by adam-ah** [comment]
@shamoon in the fix I forgot to add the need of a migration-script, to retrospectively convert existing thumbnails to webp.
